doc/source/whatsnew/v0.18.0.txt

```diff
@@ -20,6 +20,21 @@ Check the :ref:`API Changes <whatsnew_0180.api>` and :ref:`deprecations <whatsne
 New features
 ~~~~~~~~~~~~
 
+.. _whatsnew_0180.enhancements.extractall:
+
```

Comment:
put this under a separate sub-section (e.g. 'string extraction changes' or something)